cogs/wordfilter/wordfilter.py

```diff
@@ -620,11 +598,16 @@ async def orderedCensorUsageList(self, ctx):
             display = []
             pageList = []
             count = 1
-            for regex, timesUsed in dict(
-                reversed(sorted(rawUsageStats.items(), key=lambda item: item[1]))
-            ).items():
-                display.append(f"{count}. `{regex}` : `{timesUsed}`,")
-                count += 1
+            if sorting:
+                for regex, timesUsed in dict(
+                    reversed(sorted(rawUsageStats.items(), key=lambda item: item[1]))
+                ).items():
+                    display.append(f"{count}. `{regex}` : `{timesUsed}`,")
+                    count += 1
+            else:
+                for regex, timesUsed in rawUsageStats.items():
+                    display.append(f"{count}. `{regex}` : `{timesUsed}`,")
+                    count += 1
```

Comment:
We could use a separate variable to store the end result `dict` to further reduce duplicate code.

For example:
```py
stats = rawUsageStats
if sorting:
    stats = dict(sorted(rawUsageStats.items(), key=lambda item: item[1], reverse=True))
for regex, timesUsed in stats:
    display.append(f"{count}. `{regex}`: `{timesUsed}`")
    count += 1
```

Also not sure if `stats` is a common variable. Could give it another name to avoid possible ambiguity.